src/motionStudy/index.js

```diff
@@ -143,8 +184,15 @@ import {MotionStudyMobile} from './MotionStudyMobile.js'
             motionStudyByFrame[msgData.frame].hydrateMotionStudy(msgData.analyticsData);
         });
 
+        realityEditor.network.addPostMessageHandler('analyticsSetSensor', (msgData) => {
+            console.log('set sensor', msgData);
```

Comment:
Probably don't want to commit these logging messages?

Reply:
Thanks! Cleaned up throughout